.github/workflows/label_pull_request.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@v4
+      - uses: actions/labeler@v5
```

Comment:
This is insufficient for this bump.

The configuration file underwent a breaking change in formatting from v4 to v5 and the file will need to be updated.

xref: https://github.com/spacetelescope/jwst/pull/8117